static/js/browser2/util.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Functions and types shared across different components of graph browser.
+ */
+
+import _ from "lodash";
+
+export interface SourceSeries {
+  provenanceDomain: string;
+  val: { [key: string]: string };
+  importName?: string;
+  measurementMethod?: string;
+  observationPeriod?: string;
+  scalingFactor?: string;
+  unit?: string;
+}
+
+export enum PageDisplayType {
+  PLACE_STAT_VAR,
+  PLACE_WITH_WEATHER_INFO,
+  GENERAL,
+}
+
+/**
+ * Mapping for nodeTypes that need to render a special page type
+ */
+export const nodeTypeToPageDisplayTypeMapping = {
+  CensusZipCodeTabulationArea: PageDisplayType.PLACE_WITH_WEATHER_INFO,
```

Comment:
do we need a mapping  of other place types to PLACE_STAT_VAR? or add a note that it might be required as more display types are added.

Reply:
we don't need a mapping of other place types to PLACE_STAT_VAR because PLACE_STAT_VAR page type is dependent on a statVarId being provided. But yes, there may be more mappings as more display types are added